rasa/nlu/selectors/response_selector.py

```diff
@@ -323,6 +326,14 @@ def preprocess_train_data(self, training_data: TrainingData) -> RasaModelData:
 
         return model_data
 
+    def _full_response(self, label: Dict[Text, Any]) -> Optional[Dict[Text, Any]]:
+        """Given a label return the full response based on the labels id."""
```

Comment:
let's either add a full docstring or none

Reply:
I'll do so, but I am not sure this is a usefull practice: I am all with you that it is necessary for public methods. For private methods, I don't see why it is better to have no comment on a method than to add just a comment without args (assuming a helpful comment).

Reply:
I think that's something which we should discuss as part of the coding conventions. I'd argue that 
- describing parameters + return type isn't that much of an effort
- then the comment is even more helpful
- it's more consistent

Reply:
mhm, right thinking about it I agree :+1: